snooty/rstparser.py

```diff
@@ -373,10 +384,14 @@ def __call__(
         content: List[object] = [],
     ) -> Tuple[List[docutils.nodes.Node], List[docutils.nodes.Node]]:
         target, label = parse_explicit_title(text)
+        role = parse_role(rawtext)
```

Comment:
I think this is what `typ` is, actually!

Reply:
WELL THAT'S SENSIBLE